src/imodmodel/models.py

```diff
@@ -88,11 +96,49 @@ class Contour(BaseModel):
     """https://bio3d.colorado.edu/imod/doc/binspec.html"""
     header: ContourHeader
     points: np.ndarray  # pt
+    extra: List[GeneralStorage] = []
 
     class Config:
         arbitrary_types_allowed = True
 
 
+class MeshHeader(BaseModel):
+    """https://bio3d.colorado.edu/imod/doc/binspec.html"""
+    vsize: int
+    lsize: int
+    flag: int
+    time: int
+    surf: int
+
+class Mesh(BaseModel):
+    """https://bio3d.colorado.edu/imod/doc/binspec.html"""
+    header: MeshHeader
+    vertices: np.ndarray # vert
+    indices: np.ndarray # list
```

Comment:
could you please remove these comments and add description/link to description of IMOD index list to docstring before merge